### ISSUE TITLE
How to send message from seed nodes to their sampled neighbors during minibatch training?

### ISSUE THREAD

**Issue by WangXuhongCN** [dmlc/dgl#2298]
## ❓ Questions and Help
The code framework like this:
fanout = [n, n]
sampler = MultiLayerNeighborSampler(fanout, **kwargs)
collator = dgl.dataloading.EdgeCollator(g, train_eid, sampler, **kwargs)
dataloader = torch.utils.data.DataLoader(collator.dataset, collate_fn=collator.collate, **kwargs)
for batch_idx, (seed_nodes, 2_hop_graph) in enumerate(dataloader):
    
Given the **seed_nodes** and **2_hop_graph**, we can passing seeds' infomation to other nodes. After that, maybe I will replace the original node feat in g by node feats in this 2_hop_graph.

**Comment by jermainewang** [comment]
Question answered in private channel. Short answer is it cannot be implemented directly using high-level neighbor sampler and data loader APIs. One needs to customize the sampler by following the corresponding user guide section.
